docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-product-offer-management-feature-integration.md

```diff
@@ -48,16 +48,33 @@ Make sure that the following changes have been applied in transfer objects:
 
 | TRANSFER | TYPE  | EVENT   | PATH |
 | ------------- | ---- | ------ |---------------- |
-| MerchantDashboardCard      | class | Created | src/Generated/Shared/Transfer/MerchantDashboardCard      |
-| MerchantProductOfferCounts | class | Created | src/Generated/Shared/Transfer/MerchantProductOfferCounts |
+| MerchantProductOfferCounts | class | Created | src/Generated/Shared/Transfer/MerchantProductOfferCountsTransfer |
+| ProductTableCriteria.filterHasOffers | property | Created | src/Generated/Shared/Transfer/ProductTableCriteriaTransfer |
+| ProductTableCriteria.merchantReference | property | Created | src/Generated/Shared/Transfer/ProductTableCriteriaTransfer |
```

Comment:
the class can be mentioned, it doesn't belong to dependable features

Reply:
I did not understand what do you mean. Transfer `ProductTableCriteria` has gotten 2 new properties in that feature and I mentioned them here.

Reply:
Ah, now understand - both `ProductMerchantPortalGui` and `ProductOfferMerchantPortalGui` modules are not cross dependent.
Added as class